awx/settings/defaults.py

```diff
@@ -535,6 +536,7 @@ def IS_TESTING(argv=None):
 
 SOCIAL_AUTH_SAML_ORGANIZATION_ATTR = {}
 SOCIAL_AUTH_SAML_TEAM_ATTR = {}
+SOCIAL_AUTH_SAML_USER_FLAGS_BY_ATTR = {}
```

Comment:
We don't have any other `USER_FLAGS_BY_ATTR` settings, is this the only external auth that supports this so-far?

Reply:
LDAP has a similar concept with the field `LDAP User Flags by Group` (`AUTH_LDAP_1_USER_FLAGS_BY_GROUP`). Initially I was just doing attributes but I also added groups. We could change the name of the property to be more reflective of that. Maybe just shorten it to`SOCIAL_AUTH_SAML_USER_FLAGS`?